apps/quiz-for-america/quiz.js

```diff
@@ -98,6 +98,11 @@ quiz.getQuestion = function(id) {
     return id ? new Question(id) : null;
 };
 
+quiz.isAskable = function(questionNumber) {
+    var length = Object.keys(questions).length;
+    return (questionNumber > 0) && (questionNumber <= length);
+}
+
```

Comment:
This will break if we add questions with ids > 200 or remove a question.  It'll also return true for questions that don't actually exist (max is 163 right now).  How about `return questionNumber+'' in questions` to test if the questionNumber exists in the questions data?